# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/openjdk/hotfix-ppc6432.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

configure: WARNING: Ignoring value of OBJCOPY from the environment. Use command line variables instead.
checking for powerpc-t2-linux-gnu-gobjcopy... [not found]
checking for gobjcopy... [not found]
checking for powerpc-t2-linux-gnu-objcopy... [not found]
checking for objcopy... /usr/bin/objcopy

...

/usr/bin/objcopy: Unable to recognise the format of the input file `/srv/t2/src.openjdk.ppc6432.250320.105757.30180/jdk21u-jdk-21.0.3-ga/build/linux-ppc-zero-release/support/modules_libs/java.base/zero/libjvm.so'
make[3]: *** [lib/CompileJvm.gmk:148: /srv/t2/src.openjdk.ppc6432.250320.105757.30180/jdk21u-jdk-21.0.3-ga/build/linux-ppc-zero-release/support/modules_libs/java.base/zero/libjvm.so] Error 1

--- jdk21u-jdk-21.0.3-ga/make/autoconf/util_paths.m4.vanilla	2025-03-20 10:50:37.968688327 +0100
+++ jdk21u-jdk-21.0.3-ga/make/autoconf/util_paths.m4	2025-03-20 10:52:03.958772574 +0100
@@ -447,7 +447,7 @@
   if test "x$ac_tool_prefix" = x; then
     UTIL_LOOKUP_PROGS($1, $2, $3)
   else
-    prefixed_names=$(for name in $2; do echo ${ac_tool_prefix}${name} $name; done)
+    prefixed_names=$(for name in $2; do echo ${ac_tool_prefix}${name} ${ac_tool_prefix/-/64-}${name} $name; done)
     UTIL_LOOKUP_PROGS($1, $prefixed_names, $3)
   fi
 ])
